### ISSUE TITLE
Installation fails on raspberry pi running Xenial

### ISSUE THREAD

**Issue by davidvirdeforshh** [tizonia/tizonia-openmax-il#553]
OS: 16.04.4 LTS (Xenial Xerus)
----------------------------------------------

$ curl -kL https://goo.gl/Vu8qGR | bash
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   255    0   255    0     0    206      0 --:--:--  0:00:01 --:--:--   206
100   150  100   150    0     0     68      0  0:00:02  0:00:02 --:--:--   485
100  4523  100  4523    0     0   1409      0  0:00:03  0:00:03 --:--:-- 14046
DISTRIB_CODENAME=xenial
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial
[sudo] password for pi: 
Reading package lists... Done
Building dependency tree       
Reading state information... Done
build-essential is already the newest version (12.1ubuntu2).
libffi-dev is already the newest version (3.2.1-4).
apt-transport-https is already the newest version (1.2.29ubuntu0.1).
curl is already the newest version (7.47.0-1ubuntu2.11).
git is already the newest version (1:2.7.4-0ubuntu1.6).
libssl-dev is already the newest version (1.0.2g-1ubuntu4.14).
libxml2-dev is already the newest version (2.9.3+dfsg1-1ubuntu0.6).
libxslt1-dev is already the newest version (1.1.28-2.1ubuntu0.1).
python-dev is already the newest version (2.7.12-1~16.04).
0 upgraded, 0 newly installed, 0 to remove and 176 not upgraded.
N: Ignoring file 'mosquitto-Xenial' in directory '/etc/apt/sources.list.d/' as it has no filename extension
Hit:1 http://ppa.launchpad.net/flexiondotorg/minecraft/ubuntu xenial InRelease
Hit:2 http://ports.ubuntu.com xenial InRelease                                                                                     
Get:3 http://ports.ubuntu.com xenial-updates InRelease [109 kB]                                                                    
Hit:4 http://ppa.launchpad.net/nilarimogard/webupd8/ubuntu xenial InRelease                                                        
Get:5 http://apt.pilight.org stable InRelease [5,433 B]                                                                            
Hit:6 http://ppa.launchpad.net/ubuntu-mate-dev/welcome/ubuntu xenial InRelease                                                     
Get:7 http://ports.ubuntu.com xenial-security InRelease [109 kB]                                                                    
Get:8 http://ports.ubuntu.com xenial-backports InRelease [107 kB]                                   
Ign:10 https://dl.bintray.com/tizonia/ubuntu xenial InRelease                                       
Get:11 https://download.docker.com/linux/ubuntu xenial InRelease [66.2 kB]                          
Hit:12 http://ppa.launchpad.net/ubuntu-mate-dev/xenial-mate/ubuntu xenial InRelease                 
Hit:13 http://ppa.launchpad.net/ubuntu-pi-flavour-makers/ppa/ubuntu xenial InRelease                
Hit:14 http://ppa.launchpad.net/videolan/master-daily/ubuntu xenial InRelease                       
Hit:15 http://ppa.launchpad.net/videolan/stable-daily/ubuntu xenial InRelease                       
Get:16 https://dl.bintray.com/tizonia/ubuntu xenial Release [2,689 B]                               
Hit:16 https://dl.bintray.com/tizonia/ubuntu xenial Release
Ign:9 http://apt.mopidy.com stable InRelease             
Hit:17 http://apt.mopidy.com stable Release                                                                                                     
Get:18 http://ports.ubuntu.com xenial-updates/main Sources [329 kB]                                                                             
Get:19 http://ports.ubuntu.com xenial-updates/universe Sources [242 kB]                                                                         
Get:20 http://ports.ubuntu.com xenial-updates/main armhf Packages [722 kB]                                                                      
Get:21 http://ports.ubuntu.com xenial-updates/universe armhf Packages [638 kB]                                                                  
Get:22 http://ports.ubuntu.com xenial-updates/universe Translation-en [297 kB]                                                                  
Get:23 http://ports.ubuntu.com xenial-security/main Sources [141 kB]                                                                            
Get:24 http://ports.ubuntu.com xenial-security/universe Sources [95.5 kB]                                                                       
Get:25 http://ports.ubuntu.com xenial-security/main armhf Packages [447 kB]                                                                     
Get:26 http://ports.ubuntu.com xenial-security/universe armhf Packages [356 kB]                                                                 
Get:27 http://ports.ubuntu.com xenial-security/universe Translation-en [165 kB]                                                                 
Fetched 3,828 kB in 18s (208 kB/s)                                                                                                              
Reading package lists... Done
N: Ignoring file 'mosquitto-Xenial' in directory '/etc/apt/sources.list.d/' as it has no filename extension
Reading package lists... Done
Building dependency tree       
Reading state information... Done
python-pip is already the newest version (8.1.1-2ubuntu0.4).
0 upgraded, 0 newly installed, 0 to remove and 176 not upgraded.
N: Ignoring file 'mosquitto-Xenial' in directory '/etc/apt/sources.list.d/' as it has no filename extension
Collecting gmusicapi
  Downloading https://files.pythonhosted.org/packages/a1/8b/8864e8a08dc8611a7dad78534372262ac0de366bb9d7b797b109e916dd9c/gmusicapi-12.0.0.tar.gz (172kB)
    100% |████████████████████████████████| 174kB 714kB/s 
Requirement already up-to-date: soundcloud in /usr/local/lib/python2.7/dist-packages
Collecting youtube-dl
  Downloading https://files.pythonhosted.org/packages/72/35/415e01c9cf57e6aafb757237d7cd61a72bcb5c6f830b8c050858b76243a5/youtube_dl-2019.1.30.1-py2.py3-none-any.whl (1.8MB)
    100% |████████████████████████████████| 1.8MB 89kB/s 
Requirement already up-to-date: pafy in /usr/local/lib/python2.7/dist-packages
Collecting pycountry
  Downloading https://files.pythonhosted.org/packages/f8/39/04d8215fddd5b263beb36cc324e5656714a19081cacb2fc7c8e6afc2817d/pycountry-18.12.8-py2.py3-none-any.whl (10.5MB)
    100% |████████████████████████████████| 10.5MB 15kB/s 
Requirement already up-to-date: titlecase in /usr/local/lib/python2.7/dist-packages
Collecting pychromecast
  Downloading https://files.pythonhosted.org/packages/6d/8a/f92161e9859a14edc970615adda6d2015b33835006661a572e545e472de5/PyChromecast-2.4.0-py2.py3-none-any.whl
Requirement already up-to-date: plexapi in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: fuzzywuzzy in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: eventlet in /usr/local/lib/python2.7/dist-packages
Requirement already up-to-date: validictory!=0.9.2,>=0.8.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Collecting decorator>=3.3.1 (from gmusicapi)
  Downloading https://files.pythonhosted.org/packages/f1/cd/7c8240007e9716b14679bc217a1baefa4432aa30394f7e2ec40a52b1a708/decorator-4.3.2-py2.py3-none-any.whl
Collecting mutagen>=1.34 (from gmusicapi)
  Downloading https://files.pythonhosted.org/packages/30/4c/5ad1a6e1ccbcfaf6462db727989c302d9d721beedd9b09c11e6f0c7065b0/mutagen-1.42.0.tar.gz (925kB)
    100% |████████████████████████████████| 931kB 157kB/s 
Requirement already up-to-date: requests!=1.2.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.18.2,!=2.2.1,!=2.8.0,!=2.8.1,>=1.1.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: python-dateutil!=2.0,>=1.3 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: proboscis>=1.2.5.1 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: protobuf>=3.0.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: oauth2client>=1.1 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: mock>=0.7.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: appdirs>=1.1.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: gpsoauth>=0.2.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: MechanicalSoup>=0.4.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: six>=1.9.0 in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: future in /usr/local/lib/python2.7/dist-packages (from gmusicapi)
Requirement already up-to-date: simplejson>=2.0 in /usr/local/lib/python2.7/dist-packages (from soundcloud)
Requirement already up-to-date: fudge>=1.0.3 in /usr/local/lib/python2.7/dist-packages (from soundcloud)
Requirement already up-to-date: zeroconf>=0.17.7 in /usr/local/lib/python2.7/dist-packages (from pychromecast)
Requirement already up-to-date: casttube>=0.2.0 in /usr/local/lib/python2.7/dist-packages (from pychromecast)
Collecting tqdm (from plexapi)
  Downloading https://files.pythonhosted.org/packages/76/4c/103a4d3415dafc1ddfe6a6624333971756e2d3dd8c6dc0f520152855f040/tqdm-4.30.0-py2.py3-none-any.whl (47kB)
    100% |████████████████████████████████| 51kB 1.2MB/s 
Collecting websocket-client (from plexapi)
  Downloading https://files.pythonhosted.org/packages/26/2d/f749a5c82f6192d77ed061a38e02001afcba55fe8477336d26a950ab17ce/websocket_client-0.54.0-py2.py3-none-any.whl (200kB)
    100% |████████████████████████████████| 204kB 673kB/s 
Requirement already up-to-date: greenlet>=0.3 in /usr/local/lib/python2.7/dist-packages (from eventlet)
Collecting dnspython>=1.15.0 (from eventlet)
  Downloading https://files.pythonhosted.org/packages/ec/d3/3aa0e7213ef72b8585747aa0e271a9523e713813b9a20177ebe1e939deb0/dnspython-1.16.0-py2.py3-none-any.whl (188kB)
    100% |████████████████████████████████| 194kB 713kB/s 
Requirement already up-to-date: monotonic>=1.4 in /usr/local/lib/python2.7/dist-packages (from eventlet)
Requirement already up-to-date: enum34; python_version < "3.4" in /usr/local/lib/python2.7/dist-packages (from eventlet)
Requirement already up-to-date: idna<2.9,>=2.5 in /usr/local/lib/python2.7/dist-packages (from requests!=1.2.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.18.2,!=2.2.1,!=2.8.0,!=2.8.1,>=1.1.0->gmusicapi)
Requirement already up-to-date: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python2.7/dist-packages (from requests!=1.2.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.18.2,!=2.2.1,!=2.8.0,!=2.8.1,>=1.1.0->gmusicapi)
Requirement already up-to-date: certifi>=2017.4.17 in /usr/local/lib/python2.7/dist-packages (from requests!=1.2.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.18.2,!=2.2.1,!=2.8.0,!=2.8.1,>=1.1.0->gmusicapi)
Requirement already up-to-date: urllib3<1.25,>=1.21.1 in /usr/local/lib/python2.7/dist-packages (from requests!=1.2.0,!=2.12.0,!=2.12.1,!=2.12.2,!=2.18.2,!=2.2.1,!=2.8.0,!=2.8.1,>=1.1.0->gmusicapi)
Collecting setuptools (from protobuf>=3.0.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/bf/ae/a23db1762646069742cc21393833577d3fa438eecaa59d11fb04fa57fcd5/setuptools-40.7.1-py2.py3-none-any.whl (574kB)
    100% |████████████████████████████████| 583kB 258kB/s 
Requirement already up-to-date: pyasn1-modules>=0.0.5 in /usr/local/lib/python2.7/dist-packages (from oauth2client>=1.1->gmusicapi)
Requirement already up-to-date: pyasn1>=0.1.7 in /usr/local/lib/python2.7/dist-packages (from oauth2client>=1.1->gmusicapi)
Requirement already up-to-date: rsa>=3.1.4 in /usr/local/lib/python2.7/dist-packages (from oauth2client>=1.1->gmusicapi)
Collecting httplib2>=0.9.1 (from oauth2client>=1.1->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/ce/ed/803905d670b52fa0edfdd135337e545b4496c2ab3a222f1449b7256eb99f/httplib2-0.12.0.tar.gz (218kB)
    100% |████████████████████████████████| 225kB 628kB/s 
Requirement already up-to-date: funcsigs>=1; python_version < "3.3" in /usr/local/lib/python2.7/dist-packages (from mock>=0.7.0->gmusicapi)
Collecting pbr>=0.11 (from mock>=0.7.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/f3/04/fddc1c2dd75b256eda4d360024692231a2c19a0c61ad7f4a162407c1ab58/pbr-5.1.1-py2.py3-none-any.whl (106kB)
    100% |████████████████████████████████| 112kB 1.1MB/s 
Collecting pycryptodomex>=3.0 (from gpsoauth>=0.2.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/9e/ed/e44c54a648fdfa56bf0682cd94b03303fa74a70e964c86acbbedf992afbe/pycryptodomex-3.7.3.tar.gz (9.3MB)
    100% |████████████████████████████████| 9.3MB 16kB/s 
Collecting beautifulsoup4 (from MechanicalSoup>=0.4.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/8b/0e/048a2f88bc4be5e3697df9dc1f7b9d5c9c75be62676feeeb91d2e896c5ea/beautifulsoup4-4.7.1-py2-none-any.whl (94kB)
    100% |████████████████████████████████| 102kB 1.2MB/s 
Collecting lxml (from MechanicalSoup>=0.4.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/16/4a/b085a04d6dad79aa5c00c65c9b2bbcb2c6c22e5ac341e7968e0ad2c57e2f/lxml-4.3.0.tar.gz (2.5MB)
    100% |████████████████████████████████| 2.5MB 62kB/s 
Collecting ifaddr (from zeroconf>=0.17.7->pychromecast)
  Downloading https://files.pythonhosted.org/packages/9f/54/d92bda685093ebc70e2057abfa83ef1b3fb0ae2b6357262a3e19dfe96bb8/ifaddr-0.1.6.tar.gz
Requirement already up-to-date: typing; python_version < "3.5" in /usr/local/lib/python2.7/dist-packages (from zeroconf>=0.17.7->pychromecast)
Collecting soupsieve>=1.2 (from beautifulsoup4->MechanicalSoup>=0.4.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/bf/b3/2473abf05c4950c6a829ed5dcbc40d8b56d4351d15d6939c8ffb7c6b1a14/soupsieve-1.7.3-py2.py3-none-any.whl
Requirement already up-to-date: ipaddress in /usr/local/lib/python2.7/dist-packages (from ifaddr->zeroconf>=0.17.7->pychromecast)
Collecting backports.functools-lru-cache; python_version < "3" (from soupsieve>=1.2->beautifulsoup4->MechanicalSoup>=0.4.0->gmusicapi)
  Downloading https://files.pythonhosted.org/packages/03/8e/2424c0e65c4a066e28f539364deee49b6451f8fcd4f718fefa50cc3dcf48/backports.functools_lru_cache-1.5-py2.py3-none-any.whl
Building wheels for collected packages: gmusicapi, mutagen, httplib2, pycryptodomex, lxml, ifaddr
  Running setup.py bdist_wheel for gmusicapi ... done
  Stored in directory: /root/.cache/pip/wheels/33/20/8a/01a19dfc24561b24e3aa403f31112f6129d798ba19b7cda081
  Running setup.py bdist_wheel for mutagen ... done
  Stored in directory: /root/.cache/pip/wheels/33/4c/c3/6189a75038a7b00a8bc77fcb4dbdc38de335c55443f6680b13
  Running setup.py bdist_wheel for httplib2 ... done
  Stored in directory: /root/.cache/pip/wheels/6d/41/4b/2b369d6e2b7eaebcdd423516d3fb659c7658c16a2be8fd04ec
  Running setup.py bdist_wheel for pycryptodomex ... done
  Stored in directory: /root/.cache/pip/wheels/9f/ed/f8/6de93417b885f881814a35611b08ec5b4f692b45f0ef6c95a0
  Running setup.py bdist_wheel for lxml ... done
  Stored in directory: /root/.cache/pip/wheels/95/c8/60/70e98d4061350868e4a0cfc363349fa52b890a77eea72c816d
  Running setup.py bdist_wheel for ifaddr ... done
  Stored in directory: /root/.cache/pip/wheels/1a/78/2c/a5bbe977f6d91039ddc1d657e780cb55a493aa5d2c0b64637a
Successfully built gmusicapi mutagen httplib2 pycryptodomex lxml ifaddr
Installing collected packages: decorator, mutagen, gmusicapi, youtube-dl, pycountry, pychromecast, tqdm, websocket-client, dnspython, setuptools, httplib2, pbr, pycryptodomex, backports.functools-lru-cache, soupsieve, beautifulsoup4, lxml, ifaddr
  Found existing installation: decorator 4.3.0
    Uninstalling decorator-4.3.0:
      Successfully uninstalled decorator-4.3.0
  Found existing installation: mutagen 1.41.1
    Uninstalling mutagen-1.41.1:
      Successfully uninstalled mutagen-1.41.1
  Found existing installation: gmusicapi 11.0.3
    Uninstalling gmusicapi-11.0.3:
      Successfully uninstalled gmusicapi-11.0.3
  Found existing installation: youtube-dl 2018.10.29
    Uninstalling youtube-dl-2018.10.29:
      Successfully uninstalled youtube-dl-2018.10.29
  Found existing installation: pycountry 18.5.26
    Uninstalling pycountry-18.5.26:
      Successfully uninstalled pycountry-18.5.26
  Found existing installation: PyChromecast 2.3.0
    Uninstalling PyChromecast-2.3.0:
      Successfully uninstalled PyChromecast-2.3.0
  Found existing installation: tqdm 4.28.1
    Uninstalling tqdm-4.28.1:
      Successfully uninstalled tqdm-4.28.1
  Found existing installation: websocket-client 0.53.0
    Uninstalling websocket-client-0.53.0:
      Successfully uninstalled websocket-client-0.53.0
  Found existing installation: dnspython 1.15.0
    Uninstalling dnspython-1.15.0:
      Successfully uninstalled dnspython-1.15.0
  Found existing installation: setuptools 40.5.0
    Uninstalling setuptools-40.5.0:
      Successfully uninstalled setuptools-40.5.0
  Found existing installation: httplib2 0.11.3
    Uninstalling httplib2-0.11.3:
      Successfully uninstalled httplib2-0.11.3
  Found existing installation: pbr 5.1.0
    Uninstalling pbr-5.1.0:
      Successfully uninstalled pbr-5.1.0
  Found existing installation: pycryptodomex 3.7.0
    Uninstalling pycryptodomex-3.7.0:
      Successfully uninstalled pycryptodomex-3.7.0
  Found existing installation: beautifulsoup4 4.6.3
    Uninstalling beautifulsoup4-4.6.3:
      Successfully uninstalled beautifulsoup4-4.6.3
  Found existing installation: lxml 4.2.5
    Uninstalling lxml-4.2.5:
      Successfully uninstalled lxml-4.2.5
  Found existing installation: ifaddr 0.1.4
    Uninstalling ifaddr-0.1.4:
      Successfully uninstalled ifaddr-0.1.4
Successfully installed backports.functools-lru-cache-1.5 beautifulsoup4-4.7.1 decorator-4.3.2 dnspython-1.16.0 gmusicapi-12.0.0 httplib2-0.12.0 ifaddr-0.1.6 lxml-4.3.0 mutagen-1.42.0 pbr-5.1.1 pychromecast-2.4.0 pycountry-18.12.8 pycryptodomex-3.7.3 setuptools-40.7.1 soupsieve-1.7.3 tqdm-4.30.0 websocket-client-0.54.0 youtube-dl-2019.1.30.1
You are using pip version 9.0.2, however version 19.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
[sudo] password for pi: 
Collecting git+https://github.com/plamere/spotipy.git
  Cloning https://github.com/plamere/spotipy.git to /tmp/pip-ENKDN6-build
Requirement already up-to-date: requests>=2.3.0 in /usr/local/lib/python2.7/dist-packages (from spotipy==2.4.4)
Requirement already up-to-date: six>=1.10.0 in /usr/local/lib/python2.7/dist-packages (from spotipy==2.4.4)
Requirement already up-to-date: idna<2.9,>=2.5 in /usr/local/lib/python2.7/dist-packages (from requests>=2.3.0->spotipy==2.4.4)
Requirement already up-to-date: chardet<3.1.0,>=3.0.2 in /usr/local/lib/python2.7/dist-packages (from requests>=2.3.0->spotipy==2.4.4)
Requirement already up-to-date: certifi>=2017.4.17 in /usr/local/lib/python2.7/dist-packages (from requests>=2.3.0->spotipy==2.4.4)
Requirement already up-to-date: urllib3<1.25,>=1.21.1 in /usr/local/lib/python2.7/dist-packages (from requests>=2.3.0->spotipy==2.4.4)
Installing collected packages: spotipy
  Found existing installation: spotipy 2.4.4
    Uninstalling spotipy-2.4.4:
      Successfully uninstalled spotipy-2.4.4
  Running setup.py install for spotipy ... done
Successfully installed spotipy-2.4.4
You are using pip version 9.0.2, however version 19.0.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libspotify12 is already the newest version (12.1.103-0mopidy1).
0 upgraded, 0 newly installed, 0 to remove and 176 not upgraded.
N: Ignoring file 'mosquitto-Xenial' in directory '/etc/apt/sources.list.d/' as it has no filename extension
Reading package lists... Done
Building dependency tree       
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 tizonia-all : Depends: python-tizplexproxy but it is not installable
               Depends: python-tizchromecastproxy but it is not installable
               Depends: python-tizspotifyproxy but it is not installable
N: Ignoring file 'mosquitto-Xenial' in directory '/etc/apt/sources.list.d/' as it has no filename extension
E: Unable to correct problems, you have held broken packages.
Oops. Something went wrong!

**Comment by tizonia** [comment]
Hi!, thanks a lot for taking the time to report.

> The following packages have unmet dependencies:
> tizonia-all : Depends: python-tizplexproxy but it is not installable
> Depends: python-tizchromecastproxy but it is not installable
> Depends: python-tizspotifyproxy but it is not installable

Any idea of what is the dependency that is missing on your system? 

try installing them separately, e.g.:

```
sudo apt-get install python-tizplexproxy
```

**Comment by tizonia** [comment]
A couple of ideas to try:

- Update pip
> You are using pip version 9.0.2, however version 19.0.1 is available.
> You should consider upgrading via the '' command.
e.g.:
```
sudo -H pip install --upgrade pip
```

- Make sure your system is completely up-to-date:
> 0 upgraded, 0 newly installed, 0 to remove and 176 not upgraded.
```
$ sudo apt-get update && sudo apt-get upgrade && sudo apt-get dist-upgrade
```

**Comment by davidvirdeforshh** [comment]
Thanks for the tips.
I upgraded pip, did "update" and upgrade. I'm unable to do a dist-upgrade because of a bug in ubuntu mate 16.04. I'm unwilling to do a fresh install of 18.04.

After this I still get the same error message. 
Is it possible that ubuntu mate 16.04 is not supported? 
Have it worked in some older version of tizonia?

I found another user with the same symptoms here:https://www.reddit.com/r/Ubuntu/comments/8ruw07/tizonia_cloud_music_player_revamped_spotify/e0v2zhj/

(I also tried snapd and docker, but without success.)

**Comment by tizonia** [comment]
The symptoms look the same as those of the reddit user.

Tizonia is supported on vanilla 16.04, on amd64 and armhf architectures, which includes Rpi. But I have not personally tested it on Ubuntu Mate on any platform. 

>dist-upgrade because of a bug in ubuntu mate 16.04.

The fact that you can't do dist-upgrade sounds pretty serious to me. Perhaps another manifestation of an underlying problem that also affects those Tizonia python packages?.  e.g;

> E: Unable to correct problems, you have held broken packages.

Honestly, I'm doubting this is a Tizonia problem, but I'm happy to help you troubleshoot if you can provide any other info on what those held packages are.

See https://linux.die.net/man/8/apt-get:

> --ignore-hold
> Ignore package Holds. This causes apt-get to ignore a hold placed on a package. This may be useful in > conjunction with dist-upgrade to override a large number of undesired holds.

**Comment by tizonia** [comment]
@davidvirdeforshh 

I've been trying to install Ubuntu Mate 16.04.2 Xenial (from https://ubuntu-mate.org/raspberry-pi/) on my RaspberyPi 3 B without any luck at all. 

A few times (pretty much every time) the installer crashed during the installation process. Another time it appeared to have completed, but after restart the Pi, the installer showed up again (?). When I finally managed to log in, I tried to upgrade the distro (apt-get update/upgrade/dist-upgrade), this is what happened:

![img_20190210_185325](https://user-images.githubusercontent.com/3161606/52538981-2497e700-2d71-11e9-8e7b-4cba3b7ff91c.jpg)

I'm not sure what is going on but Ubuntu Mate Xenial looks really broken on the Rpi, to the point that I don't believe it is actually usable. Perhaps this has more or less worked in the past?

Based on this test, I'm thinking about closing this issue. Tizonia can't be made to work properly on a distro with this level of issues.
